src/pages/index.js

```diff
@@ -164,10 +196,24 @@ function getCardElement({ link, name }) {
   cardNameEl.textContent = name;
   cardImageEl.setAttribute("src", link);
   cardImageEl.setAttribute("alt", name + " image");
-  cardLikeBtn.addEventListener("click", () =>
-    cardLikeBtn.classList.toggle("card__like-btn_is-liked")
-  );
-  cardDeleteBtn.addEventListener("click", () => cardElement.remove());
+
+  if (isLiked) {
+    cardLikeBtn.classList.add("card__like-btn_is-liked");
+  }
+
+  cardLikeBtn.addEventListener("click", () => {
+    const isLiked = cardLikeBtn.classList.contains("card__like-btn_is-liked");
+    api
+      .changeLikeStatus(id, isLiked)
+      .then(() => cardLikeBtn.classList.toggle("card__like-btn_is-liked"))
+      .catch((err) => console.error(err));
```

Comment:
Good job that you `catch` possible errors at the end of server requests. You can simplify it with `.catch(console.error);` because `console.error` is meant to log errors